com.unity.render-pipelines.high-definition/Runtime/Material/Builtin/BuiltinData.cs

```diff
@@ -58,6 +58,9 @@ public struct BuiltinData
             public float distortionBlur;           // Define the color buffer mipmap level to use
 
             // Misc
+            [SurfaceDataAttributes("Is Lightmap")]
+            public uint isLightmap; // Currently only use with APV
```

Comment:
@FrancescoC-unity new attribute used to identify if we are a lightmap or not for APV. Encoded into the high bit of specular occlusion when APV is enabled.